man/ctags.1.rst.in

```diff
@@ -1321,17 +1321,30 @@ OPTION ITEMS
 	Limits the depth of directory recursion enabled with the ``--recurse``
 	(``-R``) option.
 
-``--options=file|directory``
-	Read additional options from file or directory. If a file is specified,
-	it should contain one option per line. If a directory is specified
-	(and scandir function is available at build configuration time), files
-	suffixed with .ctags or .conf under the directory are read. (On MSDOS or
-	MSWindows this directory traverse feature is temporary disable because
-	the contributor of this feature has no access to the platforms.
-	Volunteers are welcome). As a special case, if "--options=NONE" is
-	specified as the first option on the command line, it will disable
-	the automatic reading of any configuration options from either a file
-	or the environment (see "FILES").
+``--optlib-dir=[+]directory``
+	Adds a directory to or reset **optlib** path list.
```

Comment:
Sounds better to replace "Adds" with "Add". You could also prefix "optlib" on this line and the next with "the" if there is enough room.

Reply:
Thank you.

The descriptions of the other options use "...s" like "Defines"(--langdef) for an example.
I wonder which I should use. Any criteria?

Reply:
Seems to be a mix already used. If you leave it as "Adds", should change "reset" to "resets".

Reply:
Updated. My choice was "Add" as you suggested.